### DISCUSSION TITLE
Transmission öffnet keine HTTPS Tracker

### DISCUSSION THREAD

**Discussion by haufe-lap** [Freetz-NG/freetz-ng#198] [Allgemein]
Guten Morgen,

ich habe mir ein Image mit Freetz-NG gebaut das auch soweit sehr stabil läuft.
Leider kann ich mit Transmission nur auf einen HTTP Announce zugreifen aber nicht auf HTTPS, muss ich noch ein SSL Packet flashen?
Hab ganz vergessen zu schreiben welche Box und Transmission Version ich nutze XD
7590
Transmission 3.0
Freetz 07.21

Im Anhang noch die config von Freetz-NG
[packages.txt](https://github.com/Freetz-NG/freetz-ng/files/5980144/packages.txt)
[config.txt](https://github.com/Freetz-NG/freetz-ng/files/5980146/config.txt)

**Comment by fda77** [comment]
So wie es aussieht brauch man kein ssl-Paket wenn Transmission statisch gebaut ist: https://github.com/Freetz-NG/freetz-ng/blob/master/make/transmission/Config.in#L8
Ich könnte mir vorstellen dass transmission die ca-bundle Datei `/mod/etc/ssl/certs/ca-bundle.crt` mit den Root-Zertifikaten nicht nutzt und man diese irgendwo angeben muss

**Reply by haufe-lap** [reply]
@fda77 , danke für die Antwort.
Habe einen Befehl gefunden um dies Problem bei Linux zu beheben, dazu soll man in /etc/init.d/transmission
den Parameter ändern von 1 auf 0 : procd_set_param env TR_CURL_SSL_VERIFY=1. 
Ist es in Freetz möglich diesen zum Start des Daemons anzugeben?

**Reply by fda77** [reply]
Dies Deaktiviert wohl der Prüfung der Zertifikate. Da es aber die root-Zertifikate dazu auf der Box gibt, müsste man transmission sagen wo diese zu finden sind

Kopier die /etc/init.d/rc.transmissions mal ins home oder tmp Verzeichnis. Und dann schreib vor https://github.com/Freetz-NG/freetz-ng/blob/master/make/transmission-cgi/files/root/etc/init.d/rc.transmission#L146 ein 
`export CURL_CA_BUNDLE=/mod/etc/ssl/certs/ca-bundle.crt` dann 
`./rc.transmission restart`
Falls es klappt versuch ob es auch ohne export funktioniert. Vielleicht hat transmission auch einen Parameter mit dem man die Datei angeben kann

https://github.com/transmission/transmission/blob/master/libtransmission/web.c#L418

**Reply by haufe-lap** [reply]
Hallo,
leider besteht das Problem weiterhin das Einfügen in rc.transmission hat nichts geändert. Ein Parameter  gibt es für die Funktion auch nicht :-(.

**Reply by fda77** [reply]
Schade, sollte aber laut https://serverfault.com/a/482445
Hier https://serverfault.com/a/285739 steht dass man dem start-stop-daemon auch einen Parameter angeben kann: `--exec /usr/bin/env VAR1="Value"`
Hast du mal ins Log von transmission geshaut, falls es sowas gibt?
